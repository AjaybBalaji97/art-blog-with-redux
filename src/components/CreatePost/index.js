import React, { Component } from 'react';
import './CreatePost.css';


class CreatePost extends Component {
  handleButtonClick=(event) => {
    this.props.history.push('/posts');
  }

  render() {
    return (
      <div className="createPost">
        <button className="button" onClick={this.handleButtonClick}>CreatePost</button>


      </div>
    );
  }
}

export default CreatePost;
