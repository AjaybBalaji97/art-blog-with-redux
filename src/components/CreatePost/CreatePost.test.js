import React from 'react';
import renderer from 'react-test-renderer';
import CreatePost from '.';
import {shallow} from 'enzyme';


describe('PostContent',()=>{
    it('should render CreatePost button',()=>{
        const tree=renderer.create(<CreatePost />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});