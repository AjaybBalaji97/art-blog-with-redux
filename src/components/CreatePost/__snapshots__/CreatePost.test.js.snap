// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostContent should render CreatePost button 1`] = `
<div
  className="createPost"
>
  <button
    className="button"
  >
    CreatePost
  </button>
</div>
`;
